app/geo/gui/src/ts/twxplore/gui/geo/components/map/MapLayout.tsx

```diff
@@ -185,11 +189,21 @@ const MapLayoutImpl: React.FunctionComponent = () => {
             className={classes.drawerGridContainer}
             justify={"flex-start"}
           >
-            <Grid>
+            <Grid item xs>
               <Typography align={"center"}>Toggle Layer Type</Typography>
             </Grid>
 
-            <ToggleLayerPanel />
+            <Grid
```

Comment:
Needs to be a container?

Reply:
Line 196: I think so. ToggleLayerPanel returns item-containers which are "rows", each row representing a FeatureType and its toggle button. 
This outside item-container is to stack those rows as columns

Reply:
The outer Grid shouldn't know anything about what <ToggleLayerPanel> is doing. Components should be self-contained.